### ISSUE TITLE
Not Setting Outputs Or returning Values 

### ISSUE THREAD

**Issue by MaxPowers96** [marcoschwartz/aREST#106]
I am trying to go through your examples and trying the first one. 
//  Libraries 
#include  <SPI.h> 
#include  <aREST.h>
//  Create  aREST instance 
aREST  rest  = aREST();
void  setup(void) {  
  //  Start Serial    
  Serial.begin(115200);
    //  Give  name  and ID  to  device    
    rest.set_id("1");   
    rest.set_name("serial");
}
void  loop()  {
    //  Handle  REST  calls   
    rest.handle(Serial);
}
I am using and Arduino UNO  and when i use the serial monitor  and type  /mode/13/o I get 

{"message": "Pin D13"id": "1", "name": "serial", "hardware": "arduino", "connected": true}

it does not say that it has set it to output  and therefore when I send /digital/13/1  I get
"id": "1", "name": "serial", "hardware": "arduino", "connected": true}
 
I have no clue what I did wrong or if it is anything i did wrong, I am using all the latest libraries and version 1.6.8 arduino ide. 
Thanks in advance for any help  just getting started with this.

**Comment by marcoschwartz** [comment]
Hey Max, sorry to hear you are having this issue. Did you make sure you are using the right Serial port speed ?

**Comment by aacierno** [comment]
Hi Max, I also had the same issue where the return value was missing in the arduino serial monitor.  I tried using previous versions of the aREST library and that didn't seem to fix it.  I made sure the arduino code, the arduino serial monitor, and the xbee were all at the same speed, nothing to change there.  I happened to notice at the bottom of the serial monitor there is a window that defaults to NO LINE ENDING.  I changed that to BOTH NL AND CR and now i am getting my return value to print out.  Try that.
